tests/DockerContainerInstanceTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We don't use strict types anywhere. Could you remove this line?

Reply:
Happy to remove, I was just following the pattern you already had in `tests/DockerContainerTest.php`